src/Ng/Models/CatalogItem.cs

```diff
@@ -0,0 +1,318 @@
+        {
+            get;
+            private set;
+        }
+
+        [JsonProperty(PropertyName = "iconUrl")]
+        public string IconUrl
+        {
+            get;
+            private set;
+        }
+
+        [JsonProperty(PropertyName = "id")]
+        public string PackageId
+        {
+            get;
+            private set;
+        }
+
+        [JsonProperty(PropertyName = "isPrerelease")]
+        public bool IsPrerelease
+        {
+            get;
+            private set;
+        }
+
+        [JsonProperty(PropertyName = "language")]
+        public string Language
+        {
+            get;
+            private set;
+        }
+
+        [JsonProperty(PropertyName = "lastEdited")]
+        public DateTime LastEdited
+        {
+            get;
+            private set;
+        }
+
+        [JsonProperty(PropertyName = "packageHash")]
+        public string PackageHash
+        {
+            get;
+            private set;
+        }
+
+        [JsonProperty(PropertyName = "licenseNames")]
+        public string LicenseNames
+        {
+            get;
+            private set;
+        }
+
+        [JsonProperty(PropertyName = "licenseReportUrl")]
+        public Uri LicenseReportUrl
+        {
+            get;
+            private set;
+        }
+
+        [JsonProperty(PropertyName = "licenseUrl")]
+        public Uri LicenseUrl
+        {
+            get;
+            private set;
+        }
+
+        [JsonProperty(PropertyName = "listed")]
+        public bool Listed
+        {
+            get;
+            private set;
+        }
+
+        [JsonProperty(PropertyName = "originalId")]
+        public string OriginalId
+        {
+            get;
+            private set;
+        }
+
+        [JsonProperty(PropertyName = "packageHashAlgorithm")]
+        public string PackageHashAlgorithm
+        {
+            get;
+            private set;
+        }
+
+        [JsonProperty(PropertyName = "packageSize")]
+        public int PackageSize
+        {
+            get;
+            private set;
+        }
+
+        [JsonProperty(PropertyName = "projectUrl")]
+        public Uri ProjectUrl
+        {
+            get;
+            private set;
+        }
+
+        [JsonProperty(PropertyName = "published")]
+        public DateTime? Published
+        {
+            get;
+            private set;
+        }
+
+        [JsonProperty(PropertyName = "releaseNotes")]
+        public string ReleaseNotes
+        {
+            get;
+            private set;
+        }
+
+        [JsonProperty(PropertyName = "requireLicenseAcceptance")]
+        public bool RequireLicenseAcceptance
+        {
+            get;
+            private set;
+        }
+
+        [JsonProperty(PropertyName = "summary")]
+        public string Summary
+        {
+            get;
+            private set;
+        }
+
+        [JsonProperty(PropertyName = "verbatimVersion")]
+        public string VerbatimVersion
```

Comment:
I believe it's the package version for the v2 nuspec. i.e. they include it for backwards compatibility with the v2 APIs. I think it's always the same value as 'version'.